firmware/boot.py

```diff
@@ -0,0 +1,24 @@
+import gc
+
+import config
+import esp
+import micropython
+
+esp.osdebug(None)
+gc.collect()
+
+
+def do_connect():
+    import network
+
+    wlan = network.WLAN(network.STA_IF)
+    if not wlan.isconnected():
+        print("connecting to network...")
+        wlan.active(True)
+        wlan.connect(config.WIFI_SSID, config.WIFI_PASSWORD)
+        while not wlan.isconnected():
+            pass
```

Comment:
This is cool! So I guess it just "hangs" here until `wlan.isconnected()` and then it goes and prints out the network config?